pkg/addons/addons_test.go

```diff
@@ -119,6 +128,9 @@ func TestSetAndSave(t *testing.T) {
 	if c.Addons["dashboard"] != false {
 		t.Errorf("expected dashboard to be enabled")
 	}
+
+	// remove all temp dir
+	createTestProfile(t, false)
```

Comment:
should this be a remove call?

Reply:
@sharifelgamal yes, that is a remove call, to remove all tmp directories/files.
That's at the end, so they won't be removed before the rest of the unittests use them.